docs/api/RequestShape.md

```diff
@@ -86,3 +87,7 @@ Custom data error lifetime for the fetched resource. Will override the value set
 
 Frequency in millisecond to poll at. Requires using [useSubscription()](./useSubscription.md) to have
 an effect.
+
+#### invalidIfStale: boolean
+
+Forces useResource to suspend if the data in cache is stale. Requires `dataExpiryLength` to have effect.
```

Comment:
how about something like, "Indicates stale data should be considered unusable and thus not be returned from the cache. This means that useResource() will suspend when data is stale even if it already exists in cache."

don't want to exclude other possible uses of this